client/src/reducers/controls.js

```diff
@@ -83,10 +89,9 @@ const Controls = (
       /* first light - create world & other data-driven defaults */
       const { universe } = action;
       const world = World.createWorldFromEntireUniverse(universe);
-      const colorRGB = new Array(universe.nObs).fill(
-        parseRGB(globals.defaultCellColor)
-      );
-      const categoricalSelectionState = ControlsHelper.createCategoricalSelectionState(
+      const colorMode = null;
+      const colors = createColors(world, colorMode);
```

Comment:
Does this replace `newArray(foo).fill(parseRGB(bar))`

Reply:
After following the chain into state manager it seems like the answer is yes... this is a new abstraction

Reply:
Generally like it better than having to manually mutate here.